webots_ros2_driver/src/Driver.cpp

```diff
@@ -38,6 +40,27 @@ int main(int argc, char **argv) {
 
   std::shared_ptr<webots_ros2_driver::WebotsNode> node = std::make_shared<webots_ros2_driver::WebotsNode>(robotName);
   node->init();
+
+  // The parent process must be ros2 run. Declaring from launch file is deprecated. Remove with 2024.0.0.
+  const pid_t parentPid = getppid();
+  char buffer[4096];
+  sprintf(buffer, "/proc/%d/cmdline", parentPid);
```

Comment:
`sprintf` is reputed as non-safe and could generate compilation warnings:
```suggestion
  snprintf(buffer, sizeof(buffer), "/proc/%d/cmdline", parentPid);
```

Reply:
Yes, I should have seen this one, as I changed all of them in the Webots code recently...